x-pack/test/functional/services/monitoring/logstash_summary_status.js

```diff
@@ -7,7 +7,7 @@
 export function MonitoringLogstashSummaryStatusProvider({ getService }) {
   const testSubjects = getService('testSubjects');
 
-  const SUBJ_SUMMARY = 'logstashSummaryStatus';
+  const SUBJ_SUMMARY = 'logstashClusterStatus';
```

Comment:
Why did this need to change?

Reply:
I think the change makes it a lot clearer which summary this is for: the cluster of Logstash nodes rather than an individual node.